### DISCUSSION TITLE
Take failed task schedules into account

### DISCUSSION THREAD

**Discussion by sts-ryan-holton** [laravel/framework#50748] [Ideas]
Hi :wave:

I'm in the process of upgrading my Laravel project to Laravel 11. As such, I'm consolidating my packages that I'm using. I see Laravel provides a basic `schedule:list` [feature](https://laravel.com/docs/11.x/scheduling#defining-schedules)

Can we get the last failure date in here?

**Comment by ahinkle** [comment]
I'm not sure of a way that would support all queue drivers.  I guess you could try for something where it only supports drivers where it's available. It wouldn't be a bad idea, though for sure.
